platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/event/TestingEventBuilder.java

```diff
@@ -304,15 +303,28 @@ public TestingEventBuilder(@NonNull final Random random) {
     }
 
     /**
-     * Set the other-parent of an event
+     * Set an other-parent of an event
      * <p>
-     * If not set, an other parent will NOT be generated: the output event will have a null other parent.
+     * If not set, no other-parent will NOT be generated: the output event will have a no other-parents.
      *
      * @param otherParent the other-parent
      * @return this instance
      */
     public @NonNull TestingEventBuilder setOtherParent(@Nullable final GossipEvent otherParent) {
-        this.otherParent = otherParent;
+        this.otherParents = otherParent == null ? null : List.of(otherParent);
+        return this;
+    }
+
+    /**
+     * Set a list of other-parents of an event
+     * <p>
+     * If not set, no other-parents will NOT be generated: the output event will have a no other-parents.
```

Comment:
"no other-parent will NOT be generated"

double negative

Reply:
fixed